backend/src/index.js

```diff
@@ -22,10 +28,11 @@ io.on( 'connection', socket => {
 
 	const timestamp = new Date();
 
-	socket.on( 'setUser', ( message, reply ) => {
+	socket.on( 'setUser', async( message, reply ) => {
```

Comment:
We should optimize this function and use `getAsync` and `setAsync` only once in this file.